Sources/BraintreeCoreSwift/BTKeychain.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+import Security
+
+@objcMembers public class BTKeychain: NSObject {
+    
+    public static func setString(_ string: String, forKey key: String) -> Bool {
+        let data: Data = string.data(using: .utf8) ?? Data()
+        return setData(data, forKey: key)
+    }
+    
+    public static func stringForKey(_ key: String) -> String {
+        let data: Data = dataForKey(key) ?? Data()
+        guard let dataString = String(data: data, encoding: .utf8) else { return "" }
+        return dataString
+    }
+    
+    static func keychainForKey(_ key: String) -> String {
+        String(format: "com.braintreepayments.Braintree-API.", key)
+    }
+    
+    static func setData(_ data: Data, forKey key: String) -> Bool {
+        var success: Bool = true
```

Comment:
Do we need this success `Bool`? Could we just return true or false or an error in each place that we set it's value?

Reply:
This comes up a lot on Android. We don't have a consistent convention for single return statements in place. This could be a good start to the discussion though, there's some good [stack discussion](https://stackoverflow.com/q/36707) on the topic too.

Reply:
Yeah definitely two sides to this, just read through that stack discussion for a bit and I can see both sides on it. In the end I'm good either way! Maybe there is even a way to optimize this logic with enums so that the number of returns or bool state setting becomes less of a question/less bug prone.

Reply:
Definitely yeah there's for sure an opportunity to refactor to make things more readable. Having a single return variable in a method this long could easily lead to bugs.